src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -69,39 +69,42 @@ public function disable_v1() {
 	}
 
 	/**
-	 * Determines with backwards compatibility in mind, which template user has selected
-	 * on the Events > Settings page as their base Default template
+	 * Determines, with backwards compatibility in mind, which template user has selected
+	 * on the Events > Settings page as their base Default template.
 	 *
 	 * @since  4.9.2
+	 * @since TBD specifically pass 'event' to be clearer down the line where we check for custom templates.
 	 *
-	 * @return string Either 'event' or 'page' based templates
+	 * @return string Either 'event', 'page' or custom based templates
 	 */
 	public function get_template_setting() {
-		$template = 'event';
 		$default_value = 'default';
-		$setting = tribe_get_option( 'tribeEventsTemplate', $default_value );
+		$template = tribe_get_option( 'tribeEventsTemplate', $default_value );
 
-		if ( $default_value === $setting ) {
+		if ( empty( $template )  ) {
+			$template = 'event';
+		} elseif( $default_value === $template ) {
 			$template = 'page';
 		}
 
 		return $template;
 	}
 
 	/**
-	 * Based on the base template setting we fetch the respective object
+	 * Based on the admin template setting we fetch the respective object
 	 * to handle the inclusion of the main file.
 	 *
 	 * @since  4.9.2
+	 * @since TBD inverted logic, as all the custom templates are page templates
 	 *
 	 * @return object
 	 */
 	public function get_template_object() {
 		$setting = $this->get_template_setting();
 
-		return $setting === 'page'
-			? tribe( Template\Page::class )
-			: tribe( Template\Event::class );
+		return $setting === 'event'
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 6950f59a2bd5072b0f812b951c02ce76</sub>